src/ansys/aedt/core/visualization/advanced/rcs_visualization.py

```diff
@@ -1617,6 +1621,97 @@ def add_waterfall(
 
         self.all_scene_actors["results"]["waterfall"][waterfall_name] = rcs_mesh
 
+    @pyaedt_function_handler()
+    def add_isar_2d(
+        self,
+        plot_type="Plane",
+        color_bar="jet",
+    ):
+        """Add the ISAR 2D.
+
+        Parameters
+        ----------
+        plot_type : str, optional
+            The type of plot to create for the range profile. It can be ``"Plane"``, ``"Relief"``, and `"Projection"``.
+            The default is ``"Plane"``.
+        color_bar : str, optional
+            Color mapping to be applied to the RCS data. It can be a color (``"blue"``,
+            ``"green"``, ...) or a colormap (``"jet"``, ``"viridis"``, ...). The default is ``"jet"``.
+        """
+        # if not self.modeler_window:
+        #     self.modeler_window = BackgroundPlotter(show=False)
+
+        data_isar_2d = self.rcs_data.isar_2d
+
+        down_range = data_isar_2d["Down-range"].unique()
+        cross_range = data_isar_2d["Cross-range"].unique()
+
+        values_2d = data_isar_2d.pivot(index="Cross-range", columns="Down-range", values="Data").to_numpy()
+
+        x, y = np.meshgrid(down_range, cross_range)
+        z = np.zeros_like(x)
+
+        if plot_type.lower() == "relief":
+            # m = self.modeler_window.bounds[-1]-self.modeler_window.bounds[-2]
+            # b = self.modeler_window.bounds[-2]
+            m = 2.0
+            b = -1.0
```

Comment:
Is it expected to hard code `m` and `b` value.

Reply:
The plot in pyvista is normalized from -1 to 1 for now, so these 2 values are hardcoded in this first implementation. We will see how this method evolves, but for now is able to plot correctly the ISAR 2D Relief: 
![image](https://github.com/user-attachments/assets/28291d36-7f82-41dc-b682-cef290dabd56)

Reply:
Thanks @Samuelopez-ansys for the info!